database/mysql.go

```diff
@@ -199,7 +199,7 @@ func mysqlConnection(logger log.Logger, mysqlConfig *MySQLConfig, databaseName s
 // MySQLUniqueViolation returns true when the provided error matches the MySQL code
 // for duplicate entries (violating a unique table constraint).
 func MySQLUniqueViolation(err error) bool {
-	match := strings.Contains(err.Error(), fmt.Sprintf("Error %d: Duplicate entry", mySQLErrDuplicateKey))
+	match := strings.Contains(err.Error(), fmt.Sprintf("Error %d", mySQLErrDuplicateKey))
```

Comment:
can we do the same for `mysqlErrDataTooLong` so it's less fragile too?

Reply:
(@kolaworld beat me to it - resolving this as [dupe](https://github.com/moov-io/base/pull/265#discussion_r1038576034))

Reply:
Done